resources/public/api/conf/1.0/docs/overview.md

```diff
@@ -5,4 +5,4 @@ This API allows developers to retrieve, create, amend or delete relief investmen
 * seed enterprise investments
 * social enterprise investments.
 
-For information on how to connect to this API [see the Income Tax MTD end-to-end service guide](https://developer.service.hmrc.gov.uk/guides/income-tax-mtd-end-to-end-service-guide/#overview).
+For information on how to connect to this API [see the Income Tax MTD end-to-end service guide](https://developer.service.hmrc.gov.uk/guides/income-tax-mtd-end-to-end-service-guide/#overview)
```

Comment:
The link links to the overview section of the end to end guide instead of just the end to end guide itself